.github/workflows/haskell-ci.yml

```diff
@@ -14,24 +14,26 @@ jobs:
       matrix:
         os:
           - ubuntu-latest
-        cabal: [3.6]
+        cabal: [3.8]
         ghc:
           - 8.6.5
           - 8.8.4
           - 8.10.7
           - 9.0.2
+          - 9.2.5
+          - 9.4.4
```

Comment:
I think it'd be okay to remove 8.6.5 and 8.8.4 from CI. What do you think?

Reply:
You're right. At this point when most people are on ghc 8.10 or newer it's probably just waste of resources.